src/cli/print/print.go

```diff
@@ -89,6 +89,39 @@ func err(w io.Writer, s ...any) {
 	fmt.Fprint(w, msg...)
 }
 
+// Print prints the params to cobra's output writer (stdOut by default)
+// if s is nil, prints nothing.
+func Print(ctx context.Context, s ...any) {
```

Comment:
Not sure I'm a fan of the stutter (print.Print).  Not sure I have a good alternative, either.  What do you think of `Out` instead?
```suggestion
func Out(ctx context.Context, s ...any) {
```